src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/RequestJudgementByAdmissionNonDivergentSpecCallbackHandler.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.caseevents;
```

Comment:
Why you need this class ?

Reply:
I was told to create a handler for the new event on the last ticket, so I thought I had to do the same here.

Reply:
I think you dont require it. This handler is doing something? Is it being called by camunda tasks?